Python/Product/PythonTools/source.extension.vsixmanifest

```diff
@@ -21,7 +21,7 @@
   <Dependencies>
     <Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" Version="4.6" />
     <Dependency IsRequired="false" Version="[14.0,16.0)" d:Source="Installed" Id="TestWindow.Microsoft.0771d463-d74d-4e95-aac2-39d3c7ec1f97" DisplayName="Test Explorer" />
-    <Dependency d:Source="Installed" Id="1F42C6D0-F876-4AF0-8185-1BEB0A325BB2" DisplayName="VisualStudio Interactive Components" Version="[1.0.0.50618,2.0)" />
+    <Dependency d:Source="Installed" Id="1F42C6D0-F876-4AF0-8185-1BEB0A325BB2" DisplayName="VisualStudio Interactive Components" Version="1.0.0.0" />
```

Comment:
Actually, no, that's special handling just for VS versions. Dependency versions are treated as an exact match.

I'll just remove the version field entirely - we have programmatic handling for breakage between interactive versions already.